src/PostgREST/Middleware.hs

```diff
@@ -152,15 +151,15 @@ corsPolicy req = case lookup "origin" headers of
   where
     headers = Wai.requestHeaders req
     accHeaders = case lookup "access-control-request-headers" headers of
-      Just hdrs -> map (CI.mk . toS . T.strip . toS) $ BS.split ',' hdrs
+      Just hdrs -> map (CI.mk . BS.dropWhile (' ' ==) . BS.dropWhileEnd (' ' ==)) $ BS.split ',' hdrs
```

Comment:
Since `BS` is `Data.ByteString.Char8`, I think there is `BS.strip`, too?

https://hackage.haskell.org/package/bytestring-0.11.1.0/docs/src/Data.ByteString.Char8.html#strip

Reply:
Ah, good point! Fixed